test_unstructured_ingest/dest/sql.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e
+
+DEST_PATH=$(dirname "$(realpath "$0")")
+SCRIPT_DIR=$(dirname "$DEST_PATH")
+cd "$SCRIPT_DIR"/.. || exit 1
+OUTPUT_FOLDER_NAME=sql-dest
+OUTPUT_ROOT=${OUTPUT_ROOT:-$SCRIPT_DIR}
+OUTPUT_DIR=$OUTPUT_ROOT/structured-output/$OUTPUT_FOLDER_NAME
+WORK_DIR=$OUTPUT_ROOT/workdir/$OUTPUT_FOLDER_NAME
+CI=${CI:-"false"}
+
+# shellcheck disable=SC1091
+source "$SCRIPT_DIR"/cleanup.sh
+function cleanup {
+  # Index cleanup
+  echo "Stopping SQL DB Docker container"
+  docker-compose -f scripts/sql-test-helpers/docker-compose.yml down --remove-orphans -v
+
+
+  # Local file cleanup
+  cleanup_dir "$WORK_DIR"
+  cleanup_dir "$OUTPUT_DIR"
+  if [ "$CI" == "true" ]; then
+    cleanup_dir "$DOWNLOAD_DIR"
+  fi
+}
+
+trap cleanup EXIT
+
+# Create sql instance and create `elements` class
+echo "Creating SQL DB instance"
+# shellcheck source=/dev/null
+scripts/sql-test-helpers/create-sql-instance.sh
+wait
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+  s3 \
```

Comment:
Looks like we are using `local` instead of s3 now for destination tests to speed things up. Check out the latest 
https://github.com/Unstructured-IO/unstructured/blob/c471ea3cc7c167f1328256ef1719f6cca4849cad/test_unstructured_ingest/dest/delta-table.sh#L31